lib/main.dart

```diff
@@ -20,9 +20,8 @@ class MyApp extends StatelessWidget {
         child: MaterialApp(
           debugShowCheckedModeBanner: false,
           title: 'Flutter Demo',
-          theme: ThemeData(
-            primarySwatch: Colors.blue,
-          ),
+          theme: AppTheme.lighttheme,
+          darkTheme: AppTheme.darktheme,
```

Comment:
@AmanKumarSingh01 Use camelCase in the naming convention.

Reply:
camelCase done